### DISCUSSION TITLE
Added: LDM Notes for April 29, 2019

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2485] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-04-29.md

**Comment by DaZombieKiller** [comment]
There's a minor typo (albeit one that had me scratching my head for a brief moment, not realizing it was just a typo) near the beginning of the document: `if the base implementation that was present at compile time is removed at rune time`. `rune time` should be `run time`.

**Comment by MgSam** [comment]
@DaZombieKiller Are you sure that's a typo? I was always convinced the C# compiler had a little bit of magic in it. :)
